lxd/instance/qemu/vm_qemu.go

```diff
@@ -634,6 +634,49 @@ func (vm *Qemu) Start(stateful bool) error {
 		"-chroot", vm.Path(),
 	}
 
+	// Detect if it is possible to run as an unprivileged user.
+	var runAsUser string
+	var runAsUID int
+	for _, user := range []string{"lxd", "nobody"} {
```

Comment:
Should we put some common logic for that somewhere since I believe we're doing the exact same for dnsmasq and probably should do the same for forkdns too.

Reply:
Done